NewPass/PasswordGeneratorViewController.swift

```diff
@@ -180,23 +226,14 @@ class PasswordGeneratorViewController: UIViewController {
             self.view.layoutIfNeeded()
         }
     }
-    
+
     /// This function is used only for updating the switches to the initial default state
     private func updateSwitchesToInitialState() {
-        for attribute in viewModel.passwordAttributes {
-            switch attribute {
-            case .containsLowercaseLetters:
-                lowercaseLetterSwitch.setOn(true, animated: false)
-            case .containsUppercaseLetters:
-                uppercaseLetterSwitch.setOn(true, animated: false)
-            case .containsNumbers:
-                numberSwitch.setOn(true, animated: false)
-            case .containsSymbols:
-                symbolSwitch.setOn(true, animated: false)
-            }
+        passwordSwitches.forEach { passwordSwitch in
+            passwordSwitch.setOn(true, animated: false)
```

Comment:
This just turns on all the switches, right? If so, why not just loop over the switches themselves? (I could be missing something.)

Reply:
@jimdanger, actually, it's not obvious at first glance but if you look at the initializer in `PasswordGeneratorViewModel` you'll see that I have a default value for the attributes array and it's only 3/4 attributes, which means only 3/4 of the switches need to be enabled for the initial state. I did this just because I didn't really like that every switch was on at app launch (visual pref). However, I think I'll change this in favor of your suggestion because it's much cleaner unless you think there's a better way to keep the current functionality and clean up the code.